src/Microsoft.AspNetCore.OData/Builder/ODataConventionModelBuilder.cs

```diff
@@ -39,7 +38,17 @@ private ODataConventionModelBuilder()
         /// the functionality is due to the way assembly resolution is done in AspNet vs AspnetCore.
```

Comment:
I think these remarks are kinda vague and don't really help me understand what you're referring to here. Compatibility info here would also be useful.